client/src/app/components/stardust/Output.tsx

```diff
@@ -279,6 +286,24 @@ class Output extends Component<OutputProps, OutputState> {
         );
     }
 
+    /**
+     * Get output id.
+     * @param type The type to get the output id for.
+     * @returns output id.
+     */
+    private getOutputIdbyType(type: number) {
+        switch (type) {
+            case ALIAS_OUTPUT_TYPE:
+                return this.buildAddressForAliasOrNft();
+            case FOUNDRY_OUTPUT_TYPE:
+                return this.buildFoundyId();
+            case NFT_OUTPUT_TYPE:
+                return this.buildAddressForAliasOrNft();
+            default:
+                return "Unknown Output Id";
```

Comment:
What about Basic output type ?
![image](https://user-images.githubusercontent.com/6864498/175934822-a72e4ebd-4b58-4bdb-8bcf-065a559f0b2e.png)

Reply:
Removed that function and use output id from props.